packages/webpack/src/buildtime/assemble.js

```diff
@@ -43,7 +43,9 @@ const assembleRuntime = (KEY, runtimeModules) => {
         LAVAMOAT['${name}'] = module.exports;
       })();`
     }
-    assembly += `\n;/*${name}*/;\n${sourceString}`
+    if (sourceString) {
+      assembly += `\n;/*${name}*/;\n${sourceString}`
```

Comment:
(curiosity) is it possible for `name` to contain `*/`? if so, what happens?  is this what the semicolons are for?

Reply:
`name` is a property of a static object which is passed in here as part of an array of such objects that we hardcoded define and control ourselves (see `const runtimeChunks = [` @ `plugin.js`)